src/components/LegacyQueryEditor/VisualQueryEditor.tsx

```diff
@@ -142,7 +145,7 @@ export const VisualQueryEditor: React.FC<Props> = (props) => {
       const next = {
         ...query.expression,
         from: table,
-        where: expression,
+        where: expression as QueryEditorWhereArrayExpression,
```

Comment:
because that will require to change the component that uses this `KustoWhereEditorSection` but then I would also need to update the reference in `QueryEditorFilterSection`, then the tests... Basically all the changes I did for the non-legacy editor but I don't think it's worth it for this soon-to-be-deleted (:crossed_fingers:) code